setup.py

```diff
@@ -44,6 +44,6 @@
       long_description_content_type='text/x-rst',
       python_requires='>=3.8',
       tests_require = ['pytest', 'alchemtest'],
-      install_requires=['numpy', 'pandas>=1.4', 'pymbar>=3.0.5,<4',
+      install_requires=['numpy', 'pandas>=1.4', 'pymbar>4',
```

Comment:
>=4

Reply:
Have to do >=4. Otherwise, I will be getting
```
Processing /home/docs/checkouts/readthedocs.org/user_builds/alchemlyb/checkouts/268
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: numpy in /home/docs/checkouts/readthedocs.org/user_builds/alchemlyb/conda/268/lib/python3.11/site-packages (from alchemlyb==1.0.1+13.g52762bf.dirty) (1.23.5)
Requirement already satisfied: pandas>=1.4 in /home/docs/checkouts/readthedocs.org/user_builds/alchemlyb/conda/268/lib/python3.11/site-packages (from alchemlyb==1.0.1+13.g52762bf.dirty) (1.5.2)
ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11
ERROR: Could not find a version that satisfies the requirement pymbar==4 (from alchemlyb) (from versions: 2.0.0b0, 2.0.1b0, 2.1.0b0, 3.0.dev3, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.1.0, 4.0.1)
ERROR: No matching distribution found for pymbar==4
```